spec/factories/users.rb

```diff
@@ -52,4 +58,16 @@
       end
     end
   end
+
+  factory :admin, class: User do
```

Comment:
You can use traits for it . 
https://thoughtbot.com/blog/remove-duplication-with-factorygirls-traits